dataactcore/models/errorModels.py

```diff
@@ -46,4 +46,7 @@ class ErrorMetadata(Base):
     occurrences = Column(Integer)
     first_row = Column(Integer)
     rule_failed = Column(Text, nullable=True)
+    file_type_id = Column(Integer)
```

Comment:
The main original motivation is that the error DB is primarily used for communication between the validator and broker, recording errors to be sent back to the user, while the validation DB is primarily internal to the validator, used during the validation process.